lib/vtk/commands/socks/setup.rb

```diff
@@ -88,7 +88,7 @@ def copy_key_to_clipboard
 
         def access_request_template_url
           'https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?' \
-            'assignees=&labels=external-request%2C+operations&template=Environment-Access-Request-Template.md&' \
+            'assignees=&labels=external-request%2C+operations%2C+ops-access-request&template=Environment-Access-Request-Template.md&' \
```

Comment:
Rubocop is complaining on line length. Should change this line to something like:
'assignees=&labels=external-request%2C+operations%2C+ops-access-request&' \
and then change line 92 to:
'template=Environment-Access-Request-Template.md&title=Access+for+%5Bindividual%5D'

Reply:
Done. Thank you!